databento/historical/client.py

```diff
@@ -20,7 +20,7 @@ class Historical:
     key : str, optional
         The user API key for authentication.
         If ``None`` then the `DATABENTO_API_KEY` environment variable is used.
-    gateway : HistoricalGateway or str, default HistoricalGateway.NEAREST
+    gateway : HistoricalGateway or str, default HistoricalGateway.BO1
```

Comment:
is this the default we settled on?

Reply:
Yes, for now. As the concept of nearest for historical doesn't make sense with where we'll be hosting the data.